### ISSUE TITLE
License?

### ISSUE THREAD

**Issue by AdamFrisby** [ogxd/nanolabo-optimizer#20]
Hi - what license is this codebase under? Thanks!

**Comment by chimivari** [comment]
Normaly, it is an open-source project : 
Source : https://odgy.medium.com/mesh-decimation-done-right-95245c4b5f52

**Comment by AdamFrisby** [comment]
Yeah, but it's important to specify which one -- GitHub has a little menu in the project (or it used to) to let you quickly add one. MIT and Apache 2 are the most common these days.

Basically it's a short legal agreement that says "you can use this code", without it, copyright applies which defaults to "all rights being reserved." 

The licence will also dictate how (or if) you should be credited in products which use this code.

Another option is to put the code in the public domain (anyone can use it for any purpose with no limits), but that needs a short declaration to that effect -- the GitHub licence tool should let you do that too.

Long story short though; without a licence, users of the code are exposed to legal risk for unlicensed copyright infringements, which will stop any larger project from touching the code 

Thankyou! It looks like a really interesting library.
